test/unit_tests/aie/00_itsalive/aie.mlir

```diff
@@ -8,8 +8,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-// RUN: %PYTHON aiecc.py --unified %s
-// RUN: %PYTHON aiecc.py --no-unified %s
+// RUN: %PYTHON aiecc.py --unified --alloc-scheme basic-sequential %s
+// RUN: %PYTHON aiecc.py --no-unified --alloc-scheme basic-sequential %s
```

Comment:
Is the issue here is that the test allocates to the area reserved by the bank-aware allocator, causing conflict?  If so should we change the test (remove `address = 0`) instead of the flags?

Reply:
Ah that must be it, I wasn't sure how the bank-aware allocator was working. That works with me will change it to instead remove the address.